packages/esm-patient-allergies-app/src/index.ts

```diff
@@ -74,6 +76,16 @@ function setupOpenMRS() {
         online: { showAddAllergy: true },
         offline: { showAddAllergy: false },
       },
+      {
+        id: patientAllergiesFormWorkspace,
```

Comment:
Should the workspace also work offline? If no (i.e., if it should not be shown when offline) then all is fine, otherwise `offline: true` should be used.